tests/tests_app_examples/conftest.py

```diff
@@ -19,13 +19,16 @@ def pytest_sessionfinish(session, exitstatus):
     # TODO this isn't great. We should have each tests doing it's own cleanup
     current_process = psutil.Process()
     for child in current_process.children(recursive=True):
-        params = child.as_dict() or {}
-        cmd_lines = params.get("cmdline", [])
-        # we shouldn't kill the resource tracker from multiprocessing. If we do,
-        # `atexit` will throw as it uses resource tracker to try to clean up
-        if cmd_lines and "resource_tracker" in cmd_lines[-1]:
-            continue
-        child.kill()
+        try:
+            params = child.as_dict() or {}
+            cmd_lines = params.get("cmdline", [])
+            # we shouldn't kill the resource tracker from multiprocessing. If we do,
+            # `atexit` will throw as it uses resource tracker to try to clean up
+            if cmd_lines and "resource_tracker" in cmd_lines[-1]:
+                continue
+            child.kill()
+        except psutil.NoSuchProcess:
+            pass
```

Comment:
Could you explain this? Where is this coming from and what alternative ways to resolve this have you tried?

Reply:
It happens there is a race condition on the process being stopped. The try / catch catches those failures points.

Reply:
And do we know which tests cause this? All the command tests? Or a particular one?

Reply:
I didn't manage to reproduce it locally. But I don't believe it is an issue.